### PR TITLE
Update colors and fix splash screen

### PR DESCRIPTION
**Pull request by epicfaace**
## What does this PR accomplish?
- Update colors to a lighter blue (matching figma colors)
- Update splash screen to center logo & text

Part of #204.

## How did you test the change?
image

### CODE REVIEW

File: client/ionic/src/global.css [hunk 1]
```diff
@@ -2,3 +2,7 @@
   padding: var(--ion-safe-area-top) var(--ion-safe-area-right)
     var(--ion-safe-area-bottom) var(--ion-safe-area-left);
 }
+
+.flex > div {
+  flex: 1;
+}
```
Comment (hspinks) [inline client/ionic/src/global.css]: nit: newline at end of file

Comment (epicfaace) [inline client/ionic/src/global.css]:
```suggestion
}
```

Comment (epicfaace) [reply client/ionic/src/global.css]: How do I remove this?

Comment (SamMousa) [reply client/ionic/src/global.css]: I think you need to add it

Comment (epicfaace) [inline client/ionic/src/global.css]:
```suggestion
}

```

**Comment by hspinks** [comment]
FYI checks not marked as finished due to same issue with Netlify not notifying GitHub that the processes are complete. Looks done on Netlify. I'm going to go ahead and merge this

**Pull request merged**